src/stories/pages/GatheringDetail/GatheringIntroduce.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import GatheringIntroduce from '@/pages/GatheringDetail/GatheringIntroduce/index';
+
+const meta: Meta<typeof GatheringIntroduce> = {
+  title: 'Pages/GatheringDetail/GatheringIntroduce',
+  tags: ['autodocs'],
+  component: GatheringIntroduce,
```

Comment:
`page decorators`를 추가해주시면 화면 확인하기 더 좋을 것 같아요~
```suggestion
  component: GatheringIntroduce,
  decorators: [CommonPageLayoutDecorator],
```

Reply:
좋습니다!! 다음부터 꼭 적용하겠습니다 :)